### DISCUSSION TITLE
空投捕获器 - Solana

### DISCUSSION THREAD

**Discussion by Agyness0410** [creatorsand/.github#56] [Ideas]
#空投 #工具开发
做一个工具，将空投信息，教程，学员链接起来。
### 学习者
1. 查看近期有空投意向的项目，可按难度分级，可按所需要的技能筛选
2. 查看空投的教程
3. 加入该空投项目的社区
4. 在项目结束之后留言收获

### 贡献者
1. submit找到的空投项目的详情 -- 获得积分
2. submit空投项目的操作教程 -- 获得积分
3. 解答社区成员的操作疑问 -- 获得积分
4. 其他

### 管理者
1. review贡献者提交的空投项目，确认
2. review教程，确认
3. 社区运营
4. 空投项目关闭及状态更新

**Comment by ShirleneLiu** [comment]
缺什么角色吗？
